coalib/collecting/Collectors.py

```diff
@@ -228,6 +228,28 @@ def collect_all_bears_from_sections(sections, log_printer):
     return local_bears, global_bears
 
 
+def get_filtered_bears(language, log_printer):
+    """
+    Fetch bears and filter them based on given language.
+
+    :param language:    Filter bears based on the given language.
+    :param log_printer: The log_printer to handle logging.
+    :return:            Tuple containing dictionaries of local bears
+                        and global bears.
+    """
+    from coalib.settings.ConfigurationGathering import load_configuration
```

Comment:
Oh, was circular import the reason why I was getting an import error if I imported it with other stuff above?